src/Http/Routing/src/Microsoft.AspNetCore.Routing.csproj

```diff
@@ -13,7 +13,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;routing</PackageTags>
     <IsPackable>false</IsPackable>
-    <Nullable>enable</Nullable>
```

Comment:
And, why must `$(Nullable)` be disabled now❔ Or, is a default elsewhere making this nullable despite this removal❔

Reply:
Default is true